python/__init__.py

```diff
@@ -0,0 +1,3 @@
+#Automatically created by SCRAM
+import os
+__path__.append(os.path.dirname(os.path.abspath(__file__).rsplit('/CombineHarvester/bbHRun2Legacy/',1)[0])+'/cfipython/slc7_amd64_gcc700/CombineHarvester/bbHRun2Legacy')
```

Comment:
@PascalBaertschi did you mean to commit this file?

Reply:
No, I must have missed it